src/libs/state.js

```diff
@@ -65,12 +65,14 @@ export const workflowSelectionStore = Utils.atom({
   entities: undefined
 })
 
+/** @type {Utils.Atom<any[]>} */
```

Comment:
When an atom is created with an empty array as the initial value, TS infers its type as `Atom<never[]>`. The `never` type causes errors wherever the store's value is set. This avoids those errors by typing the atom as `Atom<any[]>` and punts on assigning stricter types.